### PR TITLE
As this is the default ‘getting started link’, clarify pre-requisites.

### PR DESCRIPTION
**Pull request by ChrisChinchilla**

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
As discussed here - https://github.com/apache/incubator-openwhisk/issues/2983#issuecomment-349264154

File: docs/actions.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Actions are stateless code snippets that run on the OpenWhisk platform. An actio
 
 Actions can be explicitly invoked, or run in response to an event. In either case, each run of an action results in an activation record that is identified by a unique activation ID. The input to an action and the result of an action are a dictionary of key-value pairs, where the key is a string and the value a valid JSON value. Actions can also be composed of calls to other actions or a defined sequence of actions.
 
+## Prerequisites
+
+You will need to use OpenWhisk CLI. Find out how to use it from with a VM installation [here](https://github.com/apache/incubator-openwhisk/blob/master/tools/vagrant/README.md#using-cli-from-outside-the-vm), or download binaries for your platform [here](https://github.com/apache/incubator-openwhisk-cli/releases).
```
Comment (rabbah) [inline docs/actions.md]: `Find out how to use it from with a VM installation` needs a rewrite/clarification.

Comment (rabbah) [reply docs/actions.md]: The CLI is also available from the edge host, `https://localhost/cli/go/download/` for local and docker for mac deployments, or the corresponding IP for docker-machine.

**Comment by rabbah** [comment]
Thanks Chris for the improvement!

**Comment by ChrisChinchilla** [comment]
@rabbah OK, try this update…

**Pull request merged**